include/eve/module/math/constant/egamma_sqr.hpp

```diff
@@ -0,0 +1,77 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/module/core.hpp>
+
+namespace eve
+{
+  //================================================================================================
+  //! @addtogroup core
+  //! @{
+  //! @var egamma_sqr
+  //!
+  //! @brief Callable object computing the egamma_sqr constant value.
```

Comment:
can you add a small latex formula. People may not have any idea what egamma_sqr means, showing the actual value approx. could be nice